yara-x/src/modules/macho/mod.rs

```diff
@@ -1599,27 +1693,29 @@ fn handle_dylib_command(
     size: usize,
     macho_file: &mut File,
 ) -> Result<(), MachoError> {
-    if size < std::mem::size_of::<DylibCommand>() {
+    // 24 bytes for all integer fields and offset in Dylib/DylibCommand
+    // fat pointer of vec makes for inaccurate count
+    if size < 24 {
+        dbg!("here, oopsie");
```

Comment:
I would remove this or at least change the message otherwise LGTM

Reply:
ah clippy didn't alert me of that, ty, removing now!